licensing/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+  - sig-release-leads
```

Comment:
Personal opinion: I don't think SIG Chairs should implicitly be subproject owners.  You have plenty of staff here